nilearn/plotting/html_stat_map.py

```diff
@@ -71,10 +71,10 @@ def _threshold_data(data, threshold=None):
     # Mask data
     if threshold == 0:
         mask = (data == 0)
-        data = data * mask
+        data = data * (~mask)
```

Comment:
It's pedantic, but I prefer to use np.logical_not, because I find it more explicit than "~"